packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -109,6 +111,11 @@ contract StableToken is
     require(inflationRate != 0, "Must provide a non-zero inflation rate.");
 
     _transferOwnership(msg.sender);
+
+    // At network launch, stable token transfers should be frozen.
+    frozen = true;
+    setFreezer(msg.sender);
```

Comment:
mmm... not sure about this.

I think this is `validator0`... but we then transfer ownership to the governance contract. Probably freezer should also be the owner... in fact it's probably not necessary to have a duplicated field for taht

Reply:
I'm reusing the `Freezable` contract we already had defined for other functionalities, where we did want the owner and freezer to be different parties. 

At contract deployment/proxy initialization, all control is given to validator0, which is then transferred to Governance in one of the last migrations. This PR adds code to also transfer `freezer` to Governance at that point.

Reply:
I think it's more "clean" if we keep owner and freezer as the same. So instead of making the migration responsible of doing that, modify `setOwner()` to also call `setFreezer()`.

That way; the semantic rules are internal to the contract, and not dependent of the migration code